contracts/ref/cloud.yml

```diff
@@ -6566,12 +6615,50 @@ paths:
         "400":
           $ref: '#/components/responses/GeneralServerError'
           description: Invalid request
+        "401":
+          $ref: '#/components/responses/AuthorizationError'
+        "500":
+          $ref: '#/components/responses/InternalServerError'
```

Comment:
should responses typically have text like this: $ref: '#/components/responses/InternalServerError'

Noticed a couple $ref: '#/components/responses/401' and $ref: '#/components/responses/500' elsewhere

Reply:
Yes, these are references to other sections in the contracts. Using `$ref` lets you "modularize" your content--write it once, use it anywhere. OpenAPI viewers (Redocly) resolve the references to the sections and render the content.